egs/wsj/s5/utils/prepare_lang.sh

```diff
@@ -334,6 +357,21 @@ cat $tmpdir/lexiconp.txt | awk '{print $1}' | sort | uniq  | awk '
     printf("</s> %d\n", NR+3);
   }' > $dir/words.txt || exit 1;
 
+# In case there are extra word-level disambiguation symbols they also
+# need to be added to words.txt
+if [[ ! -z $extra_word_disambig_syms ]]; then
+  # Since words.txt already exist, we need to extract the current word count.
+  word_count=`tail -n 1 $dir/words.txt | awk '{ print $2 }'`
+
+  # We expect a file containing valid word-level disambiguation symbols.
+  # The regular expression for awk is just a paranoia filter (e.g. for empty lines).
+  cat $extra_word_disambig_syms | awk -v WC=$word_count '
+  /^#\w+$/
```

Comment:
I think it would be better instead of filtering out empty lines, to make sure that the validation script rejects files with empty lines.  Also, please use less vertical space in this awk script, you could do it like
```
 cat $extra_word_disambig_syms | \
    awk  { .. all one one line }  
```
which is more similar to existing scripts.